transaction_gateway.go

```diff
@@ -42,24 +42,6 @@ func (g *TransactionGateway) SubmitForSettlement(id string, amount ...*Decimal)
 	return nil, &invalidResponseError{resp}
 }
 
-// Settle settles a transaction.
-// This action is only available in the sandbox environment.
-func (g *TransactionGateway) Settle(id string) (*Transaction, error) {
```

Comment:
I think we should keep this function because others may be using it, but have it call out to `g.Testing().Settle(id)`, and add a note to the last line of the comments above this function that say `// Deprecated: use the Settle function on the TestingGateway instead. e.g. g.Testing().Settle(id)`. Then we'll remove this function in the next major version.

The deprecated comment format comes from: https://blog.golang.org/godoc-documenting-go-code